includes/controllers/class.llms.controller.orders.php

```diff
@@ -304,6 +310,24 @@ public function error_order( $order ) {
 
 	}
 
+	/**
+	 * Called when a post is permanently deleted.
+	 * Will delete any enrollment records linked to the LLMS_Order with the ID of the deleted post
+	 *
+	 * @since [version]
+	 *
+	 * @param int $post_id WordPress post ID
+	 * @return void
+	 */
+	public function on_delete_order( $post_id ) {
+
+		$order = new LLMS_Order( $post_id );
```

Comment:
it'll be safer to use `llms_get_post()` here.

I'd write it like this:
```
$order = llms_get_post( $post_id );
if ( $order && is_a( 'LLMS_Order', $order ) ) {
...
```

Reply:
fine, thanks.

fixed in e75d820